auto_submit/lib/service/validation_service.dart

```diff
@@ -368,6 +372,41 @@ Exception: ${exception.message}
     return ProcessMergeResult.noMessage(true);
   }
 
+  /// A user can provide an alternate merge method via comment by providing the
+  /// string @autosubmit:<merge_method>.
```

Comment:
Can you add more to this docstring, outlining that this function looks for the latest occurrence of the regex, and returns that, or, returns the default method if no comment was found. It will help people in future on what the expectation of this function is.

Reply:
Yup, good call.